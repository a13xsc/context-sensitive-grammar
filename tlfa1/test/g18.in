({T,S,C,Z,E,D,R,Q,X,L,P,F,N,W,I,B,K,U,J,G,A,M,Y,H,O,V},{},{(A,e),(C,e),(B,SVWIZUAIIZUIM),(D,e),(I,UWAUUWMAMWMUUWUMWMMU),(K,e),(J,e),(M,WWAWAWWAAAWA),(L,e),(O,e),(N,e),(S,AMIYMZMMAAIII),(U,WMMAWWAAMWAWW),(T,e),(W,AAAAAAAAAAA),(V,MU),(V,SWMCAI),(Y,MIUIZIZZUIWAUWMIM),(X,e),(Z,UWWMAWAMIMIUWWWMUU)},Q)
