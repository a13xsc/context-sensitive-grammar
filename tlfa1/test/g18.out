E
H
G
Q
F
P
R
